### PR TITLE
Add redirect example

### PR DESCRIPTION
**Pull request by sergiodxa**
Based on https://github.com/now-examples/redirect.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ These deployments:
 | [node-http-microservice-pkg](https://node-http-microservice-pkg-v2.now.sh) | [Node.js](https://en.wikipedia.org/wiki/Node.js) | A microservice in Node, packaged with [pkg](https://github.com/zeit/pkg) | 19.3M |
 | [node-micro](https://node-micro-v2.now.sh) | [Node.js](https://en.wikipedia.org/wiki/Node.js) | A microservice in Node written with [micro](https://github.com/zeit/micro) | 19.6M |
 | [node-react](https://node-react-v2.now.sh) | [Node.js](https://en.wikipedia.org/wiki/Node.js) | Deploys a react app created using [create-react-app](https://github.com/facebook/create-react-app) | 700K |
+| [node-redirect](https://node-redirect.now.sh) | [Node.js](https://en.wikipedia.org/wiki/Node.js) | A simple microservice to setup a redirect | 44.1M |
```
Comment (pranaygp) [inline README.md]: This is definitely a case for `misc-`

**Comment by sergiodxa** [comment]
Will migrate it to Go, not ready to be merged yet

File: README.md [hunk 2]
```diff
@@ -35,6 +35,7 @@ These deployments:
 | [java-spark-graal](https://java-spark-graal-v2.now.sh) | [Java](https://en.wikipedia.org/wiki/Java_(programming_language)) | Uses the [`Spark`](http://sparkjava.com/) framework and packages with [GraalVM `native-image`](http://www.graalvm.org/) | 5.3M |
 | [misc-chat-roulette](https://misc-chat-roulette-v2.now.sh) | [Node.js](https://en.wikipedia.org/wiki/Node.js) | Implements a [WebSocket](https://en.wikipedia.org/wiki/WebSocket) server with a max concurrency of `2` | 19.3M |
 | [misc-dump](https://misc-dump-v2.now.sh) | [Node.js](https://en.wikipedia.org/wiki/Node.js) | An app that simply logs information about incoming requests and the system it's running on | 26.1M |
+| [misc-redirect](https://misc-redirect.now.sh) | [Node.js](https://en.wikipedia.org/wiki/Node.js) | A simple microservice to setup a redirect | 44.1M |
```
Comment (TooTallNate) [inline README.md]: 44mb :( Need to rewrite this in bashttpd

Comment (pranaygp) [reply README.md]: 👍 @TooTallNate

Comment (pranaygp) [reply README.md]: or even asmhttpd

Comment (TooTallNate) [reply README.md]: For real though!

**Pull request merged**